skills/sosreport-analyzer/scripts/analyze.sh

```
@@ -0,0 +1,441 @@
+        add_issue "MEDIUM" "performance" \
+            "CPU iowait 偏高 (${avg_iowait}%)" \
+            "监控磁盘 I/O 性能"
+    fi
+}
+
+# 分析失败的服务
+analyze_failed_services() {
+    local failed="${REPORT_DIR}/services/systemctl-failed.txt"
+    [ ! -f "$failed" ] && return
+
+    local failed_count=$(grep -c "●" "$failed" 2>/dev/null || echo 0)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)